.ci/workspace-startup/workspace-startup-test-ephemeral.groovy

```diff
@@ -40,9 +40,10 @@ pipeline {
                 }
                 dir ("${LOG_DIR}") {
                     silent_sh """
+                    wget https://che-devfile-registry.prod-preview.openshift.io/devfiles/nodejs/devfile.yaml -O ../${RELATIVE_PATH}/nodejs_eph.yaml
```

Comment:
@ScrewTSW why do you use prod-preview specific devfile?

Reply:
because the tests are running against prod-preview. so it makes sense to use it's devfile registry

Reply:
can we use getting-started devfile instead e.g.  https://github.com/redhat-developer/devfile/tree/master/getting-started/nodejs - https://www.eclipse.org/che/getting-started/cloud/

This one must work for both prod & prod-preview

Reply:
It was recommended against by @rhopp and I understand where he's coming from. This way we are absolutely sure that the devfiles we use should be working with the version of Che they are running against and this process is automatic, no manual steps required.

Reply:
not following, we have absolutely no tests for getting-started devfiles, why not to have at least startup tests running against them?

Reply:
@rhopp ^ what do you think?

Reply:
@ScrewTSW OK, so let's keep stacks from prod and prod-preview here. But please create an issue about testing stacks on getting started page so we will not forget that. @ibuziuk Are you OK with that?

Reply:
> So basically you are saying that devfiles from getting-started should work both on production and prod-preview so there should be no problem using them. And as a value-added, we will have at least one of them tested. Am I right? 

correct

> I wouldn't necessarily mix this together. Workspace startup tests are supposed to test "known good" stacks and collect data about startup times.

wdym, "known good"? if getting-started devfiles do not work we are screwed. Those devfiles  
even more important than UD devfiles since it is first user experience

Reply:
@ibuziuk I still think that we should have a separate test suite and jenkins jobs for the getting started ones.
By "known good" I meant that the devfiles we have in registries should be tested by upstream and provided in a working form.